scripts/jenkins/ardana/ansible/run-crowbar-tempest.yml

```diff
@@ -0,0 +1,65 @@
+#
+# (c) Copyright 2018 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+---
+
+- name: Run ardana tempest and collect results
```

Comment:
"ardana" shouldn't be there. But more generally, can we combine the run-crowbar-tempest and run-ardana-tempest files? The only difference seems to be that ardana prefixes the hosts with "ardana", but the prefix can just be a variable.